### PR TITLE
Migration to Flink 1.9

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/FlinkStreamingProcessRegistrar.scala [hunk 1]
```diff
@@ -70,18 +70,18 @@ class FlinkStreamingProcessRegistrar(compileProcess: (EspProcess, ProcessVersion
     usingRightClassloader(env) {
       register(env, compileProcess(process, processVersion), testRunId)
     }
-    initializeStateDescriptors(env)
+    //initializeStateDescriptors(env)
```
Comment (arkadius) [inline engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/FlinkStreamingProcessRegistrar.scala]: why you commented it?

File: engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkStubbedRunner.scala [hunk 2]
```diff
@@ -37,19 +37,25 @@ trait FlinkStubbedRunner {
     val configuration: Configuration = new Configuration
     configuration.addAll(jobGraph.getJobConfiguration)
     configuration.setInteger(TaskManagerOptions.NUM_TASK_SLOTS, env.getParallelism)
+
+    configuration.setInteger(JobManagerOptions.PORT, 0)
```
Comment (arkadius) [inline engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkStubbedRunner.scala]: Why it need to be set? What will happen after that?

File: engine/kafka-test-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaClient.scala [hunk 3]
```diff
@@ -16,17 +19,17 @@ class KafkaClient(kafkaAddress: String, zkAddress: String, id: String) {
 
   private val consumers = collection.mutable.HashSet[KafkaConsumer[Array[Byte], Array[Byte]]]()
 
-  private val zkClient = ZkUtils.createZkClient(zkAddress, 10000, 10000)
-  private val zkUtils = ZkUtils(zkClient, isZkSecurityEnabled = false)
+  private val zkClient = KafkaZkClient(zkAddress, false, 10000, 10000, 100, Time.SYSTEM, "", "")
```
Comment (arkadius) [inline engine/kafka-test-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaClient.scala]: can u use named params?

File: engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/FlinkStreamingProcessRegistrar.scala (hunk 1 above)
Comment (mproch) [reply engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/FlinkStreamingProcessRegistrar.scala]: I wonder if it's still valid... I added TODO for now...

File: engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkStubbedRunner.scala (hunk 2 above)
Comment (mproch) [reply engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkStubbedRunner.scala]: good question... I removed it and tests still pass...

File: engine/kafka-test-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaClient.scala (hunk 3 above)
Comment (mproch) [reply engine/kafka-test-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaClient.scala]: done

**Pull request merged**